src/makeRobot.js

```diff
@@ -38,7 +37,58 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const Robot = {
+    name,
+    wheels,
+    version,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+
+    get info() {
+      return [
+        `name: ${this.name}`,
+        `chip version: ${this.version}`,
+        `wheels: ${this.wheels}`,
+      ].join(', ');
+    },
+
+    get location() {
+      return `${this.name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+
+    goForward(y = 1) {
+      this.coords.y += y < 0 ? 0 : y;
```

Comment:
I understand that it's a simple ternary, but you should have a habit of formatting ternaries properly, so fix it everywhere
```suggestion
      this.coords.y += y < 0 
        ? 0 
        : y;
```

Reply:
I'd also say that you don't need to add `0` here)